.github/workflows/feg-workflow.yml

```diff
@@ -0,0 +1,148 @@
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - orc8r/**
+      - lte/**
+      - feg/**
+  pull_request:
+    branches:
+      - master
+    paths:
+      - orc8r/**
+      - lte/**
+      - feg/**
+jobs:
+  feg-lint:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on  # yamllint disable-line rule:truthy
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run golang_before_install.sh script
+        run: ./circleci/golang_before_install.sh
+      - name: Run go mod download with retry
+        uses: nick-invision/retry@v2
+        with:
+          command: cd ${MAGMA_ROOT}/cwf/gateway && go mod download
+          timeout_minutes: 10
+      - name: Go lint code
+        run: |
+              cd ${MAGMA_ROOT}/feg/gateway
+              make -C ${MAGMA_ROOT}/feg/gateway lint
+      - name: Generate test coverage
+        run: |
+              cd ${MAGMA_ROOT}/feg/gateway
+              make -C ${MAGMA_ROOT}/feg/gateway cover
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action feg-lint update failed"
+          SLACK_USERNAME: "Feg workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  feg-precommit:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on  # yamllint disable-line rule:truthy
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run golang_before_install.sh script
+        run: ./circleci/golang_before_install.sh
+      - name: Run go mod download with retry
+        uses: nick-invision/retry@v2
+        with:
+          command: cd ${MAGMA_ROOT}/cwf/gateway && go mod download
+          timeout_minutes: 10
+      - name: make feg precommit
+        run: |
+            cd ${MAGMA_ROOT}/feg/gateway
+            make -C ${MAGMA_ROOT}/feg/gateway precommit
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
+        if: always()
+        with:
+          files: /tmp/test-results
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action feg-precommit update failed"
+          SLACK_USERNAME: "Feg workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  feg-build:
+    needs: feg-precommit
+    runs-on: ubuntu-latest
```

Comment:
add this to avoid uploading builds from PRs: `if: github.ref == 'refs/heads/master'`

Reply:
Sorry to go back and forth on this but could we trigger the build part with pull request event but just not the push and notification.
TLDR: put the if statement on the tag&push and notifications steps rather than here

Reply:
I agree.. like we did in the other workflow... thank you!

Reply:
You can remove the if statement from there now